pom.xml

```diff
@@ -50,7 +50,7 @@
         <dep.airlift.version>0.148</dep.airlift.version>
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.slice.version>0.29</dep.slice.version>
-        <dep.aws-sdk.version>1.11.30</dep.aws-sdk.version>
+        <dep.aws-sdk.version>1.11.165</dep.aws-sdk.version>
```

Comment:
Should I update version to the latest or leave as it is? What do you folks think?
cc: @nezihyigitbasi @electrum